keyboards/flx/lodestone/lodestone.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 Shaun Mitchell (Flex)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+// standard 65% layout ANSI + ISO
+
+#define LAYOUT( \
```

Comment:
If the PCB supports more than one physical layout, and the layout currently depicted supports every possible switch location, this should be named `LAYOUT_all`.

Reply:
Updated this where required, and re-tested on my hardware to confirm it still works.

Thank you for your time.